modules/private/mail.go

```diff
@@ -49,5 +49,10 @@ func SendEmail(subject, message string, to []string) (int, string) {
 		return http.StatusInternalServerError, fmt.Sprintf("Response body error: %v", err.Error())
 	}
 
-	return http.StatusOK, fmt.Sprintf("Was sent %s from %d", body, len(to))
+	var users = fmt.Sprintf("%d", len(to))
+	if len(to) == 0 {
+		users = "all"
+	}
+
+	return http.StatusOK, fmt.Sprintf("Was sent %s from %s", body, users)
```

Comment:
I get this as a response
```log
Success: Was sent 12 from all
```
![image](https://user-images.githubusercontent.com/61180606/97183679-5ac40700-179e-11eb-879b-b2df2c6ddaa2.png)
which maybe should have been something like `"Sent 12 emails in total to all users"`

Reply:
```suggestion
	return http.StatusOK, fmt.Sprintf("Sent %s email(s) to %s users", body, users)
```

Reply:
yep, this...cheers @techknowlogick @lunny